firefox-ios/firefox-ios-tests/Tests/ClientTests/SummarizerServiceTests.swift

```diff
@@ -0,0 +1,110 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+@testable import Client
+import XCTest
+import WebKit
+
+final class SummarizerServiceTests: XCTestCase {
+    static let mockResponse = ["Summarized", "content"]
+    static let maxWords = 100
+    static let maxWordCount = 50
+    var mockWebView = WKWebViewMock(URL(string: "https://foo.com")!)
+
+    func testSummarizerServiceReturnsSummary() async throws {
+        let subject = createSubject()
+        let result = try await subject.summarize(from: mockWebView)
+        XCTAssertEqual(result, Self.mockResponse.joined(separator: " "))
```

Comment:
sometimes for tests, its easier to read the assertions when we keep the strings explicit. This is my personal preference so no strong opinions here. cc: @yoanarios 
```suggestion
        XCTAssertEqual(result, "Summarized content")
```